client/src/pages/AddFood.js

```diff
@@ -42,27 +43,26 @@ function AddFood() {
   return (
     <div className="form">
       <Layout />
-      <Form onSubmit={onSubmit}>
+      <Form onSubmit={handleSubmit((data) => setData(JSON.stringify(data)))}>
         <h2>Add a New Food Item </h2>
         <label>Food Name</label>
         <input
           type="text"
-          name="food"
-          {...register("food")}
-          onChange={handleChange}
+          {...register("foodName")}
+          placeholder="Food name"
           value={food}
```

Comment:
Should this `value` be removed now that the field is being managed by `react-hook-form`?